library/jsonviewer/src/main/java/org/billcarsonfr/jsonviewer/ValueItem.kt

```diff
@@ -77,10 +77,7 @@ internal abstract class ValueItem : EpoxyModelWithHolder<ValueItem.Holder>() {
             menuInfo: ContextMenu.ContextMenuInfo?
         ) {
             if (copyValue != null) {
-                val menuItem = menu?.add(
-                    Menu.NONE, R.id.copy_value,
-                    Menu.NONE, R.string.copy_value
-                )
+                val menuItem = menu?.add(R.string.copy_value)
```

Comment:
Note for myself: test that this is still working

Reply:
Ok, it's working fine:

<img width="383" alt="image" src="https://user-images.githubusercontent.com/3940906/157407482-8d2acf0b-2cf0-4dda-b08f-695942805342.png">

and when clicked the text is correctly copied.